package.json

```diff
@@ -16,11 +16,9 @@
     "@nuxtjs/axios": "^5.3.6",
     "@nuxtjs/dotenv": "^1.4.0",
     "@nuxtjs/pwa": "^3.0.0-0",
-    "bootstrap": "^4.4.1",
+    "bootstrap": "^4.1.3",
```

Comment:
Is there a  reason this has been downgraded?

Reply:
nme run na latest ina brick

Reply:
I'll upgrade that in a few though it has no effect